pkgs/applications/version-management/gitless/default.nix

```diff
@@ -0,0 +1,31 @@
+{ fetchFromGitHub, pythonPackages, stdenv, makeWrapper, git }:
+
+pythonPackages.buildPythonApplication rec {
+  ver = "0.8.5";
+  name = "gitless-${ver}";
+
+  src = fetchFromGitHub {
+    owner = "sdg-mit";
+    repo = "gitless";
+    rev = "v${ver}";
+    sha256 = "1v22i5lardswpqb6vxjgwra3ac8652qyajbijfj18vlkhajz78hq";
+  };
+
+  postFixup = ''
+    wrapProgram $out/bin/gl --prefix PATH : ${git}/bin
+  '';
+
+  propagatedBuildInputs = (with pythonPackages; [ sh pygit2 clint ]) ++ [ git ];
```

Comment:
I expect `git` is unnecessary here, if it is merely to ensure its availability at runtime (accomplished by the wrapper alone).

Reply:
Updated. I thought I needed it previously as it was failing during testing. It turns out it's due to gitless not knowing how to deal with headless repos.